tools/NamedPipeCmd/main.go

```diff
@@ -0,0 +1,142 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Since this folder is highly specific to windows. Could you modify the CODEOWNER file to make your team owner of that folder? Our review for modification on that file is probably not very relevant